index.js

```diff
@@ -51,7 +57,7 @@ function TrvsCov(runner) {
           }
 
           if (userOpts){
-            options.threshold = userOpts[THRESHOLD_KEY] || options.threshold;
+            options.threshold = options.threshold || userOpts[THRESHOLD_KEY];
```

Comment:
Note that the order switch here allows the environment variable to take precedence over the "package.json" file's configuration values (if any). Also, you shouldn't worry about the order being opposite of the 2 lines below this line since those options are both obsolete (see PR #8).